readthedocs/projects/tasks.py

```diff
@@ -909,7 +909,7 @@ def remove_dir(path):
     shutil.rmtree(path, ignore_errors=True)
 
 
-@task(queue='web')
+@task()
 def clear_artifacts(version_pk):
     """Remove artifacts from the web servers"""
```

Comment:
It will still only work on the web servers, but instead run on the `web0[1-4]` queues instead of the generic web queue.